_posts/2024-01-17-January.md

```diff
@@ -0,0 +1,187 @@
+
+### Shadow Forest
+
+{% include figure.html src="/static/blog/2024_EarlyWinter/shadowForest.png" caption='The shadows are coming!' %}
+
+Shadow Forest is a polished Role Playing game, where you are a wizard with a tight orb companion who follows you everywhere. It is your job to rid the forest of shadows which have invaded it. Use your weapons to defeat enemies, but don't become too caught up in battle and stall your progress through the world.
+
+### Air Game
+
+by wsor
+
+*this section left intentionally blank*
+
+
+## Mods News
+
+### Automobiles Pack
+
+{% include figure.html src="/static/blog/2024_EarlyWinter/vwbug.png" caption='Lovebug for Minetest.' %}
+
+Apercy added a beetle car to his Automobiles Pack mod.
+
+### TypeScript For Minetest
+
+jordan4ibanez has created a Typescript modding system on top of the lua api using the TypeScriptToLua complier. He make a custom TypeScript typing system specifically for Minetest, and is using it in his project ForgottenLands.
+
+## Art and Builds
+
+### Digilines Computer
+
+{% include figure.html src="/static/blog/2024_EarlyWinter/digilines_comp.png" caption=' The ventilated computer with several commands executed on its large screenn, by Bituvo. The digilines computer can be found on the server "Eden Lost". From spawn, go to the "travelnets" section, and go to the first travelnet on the right labelled "Thresher". Visit the "Computer" station.
+' %}
+
+
+## Server News
+
+
+## Other News
+
+### Minetest at Events in December
+
+In late 2023, Minetest was featured at several conventions around Italy thanks
+to Zughy and his sponsor Italian Linux Society. Zughy has brought Minetest to
+several events in Northern and Central Italy from underground videogame
+festivals like [Zona Warpa](https://www.zonawarpa.it/) to Free software events like [MERGE-it](https://merge-it.net/). He demoed games
+and gave talks (you can listen one of them [here](https://www.youtube.com/watch?v=KBZoPYleoX4), in spaghetti language 🍝 ). In
+December Zughy also presented a workshop at Empoli for the first time, where
+participants learned how to create a complete minigame from scratch using
+[arena_lib](https://content.minetest.net/packages/Zughy/arena_lib/). It doesn't seem like these events are going to stop. What's next?
+FOSDEM!
+
+## FOSDEM 2024
+
+Minetest will be at FOSDEM 2024. FOSDEM is the largest conference in Europe for
+free and open source software, and will be held from 3-4th February in Brussels,
+Belgium. We'll be running a booth, showing off Minetest, giving out stickers and
+goodies, and introducing visitors to the Minetest!
+
+If you happen to be in the area, come out to show your support!
```

Comment:
Come show your support, sounds weird. Are you accepting donations? Maybe come say hi!